tg-portfolio/components/Navbar.jsx

```diff
@@ -1,9 +1,70 @@
-import React from 'react'
+import React from 'react';
+import Link from 'next/link';
 
 const Navbar = () => {
+  console.log('Navbar');
+  // const btn = document.querySelector('button.mobile-menu-button');
+  // const menu = document.querySelector('.mobile-menu');
   return (
-    <div>Navbar</div>
-  )
-}
+    // Navbar
+    <nav className=" bg-cyan-400/75 shadow-lg">
+      <div className="max-w-10xl mx-auto px-48 md:px-16">
+        <div className="flex justify-between">
+          <div className="flex lg:space-x-7">
+            <div />
+            {/* Website Logo or Title */}
+            <Link href="/">
+              <div className="text-xl font-extrabold flex items-center py-4 px-2">
+                <h1 className=""><span className="hover:text-zinc-500 cursor-pointer hover:animate-pulse">Travis Geislinger </span>
+                </h1>
+              </div>
+            </Link>
+          </div>
+          <div className="hidden md:flex items-center space-x-11">
+            <Link href="/">
+              <p className="cursor-pointer py-4 px-2 border-b-4 border-zinc-500 font-semibold
+              hover:animate-pulse hover:text-zinc-800 navName"
+              >About
+              </p>
+            </Link>
+            <Link href="/">
+              <p className="cursor-pointer py-4 px-2 border-b-4 border-zinc-500 font-semibold
+              hover:animate-pulse hover:text-zinc-800 navName"
+              >Services
+              </p>
+            </Link>
+            <Link href="/">
+              <p className="cursor-pointer py-4 px-2 border-b-4 border-zinc-500 font-semibold
+              hover:animate-pulse hover:text-zinc-800 navName"
+              >Porfolio
+              </p>
+            </Link>
+            <Link href="/">
+              <p className="cursor-pointer py-4 px-2 border-b-4 border-zinc-500 font-semibold
+              hover:animate-pulse hover:text-zinc-800 navName"
+              >Contact
+              </p>
+            </Link>
```

Comment:
These Links are essentially the same except the name & URL. Instead of repeating same structure/code, create a navLinks object and loop over a single structure. 

```javascript
const navLinks = [
  {
     id: 'nav-link-1',
     name: "About",
    link: "/"
  },
  {},
  {},
  ...
]

navLinks.map((item) => (
<Link href={item.link}>
  ....
 </Link>
))
```